content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -442,7 +442,7 @@ topology spread constraints are applied to a Pod if, and only if:
 
 Default constraints can be set as part of the `PodTopologySpread` plugin
 arguments in a [scheduling profile](/docs/reference/scheduling/config/#profiles).
-The constraints are specified with the same [API above](#api), except that
+The constraints are specified with the same [API above](#example-topologyspreadconstraints-with-nodeaffinity), except that
```

Comment:
Would it make sense to point to the #topologyspreadconstraints-field instead?

Reply:
Yes @annajung we can replace it with #topologyspreadconstraints-field , should I proceed with changes?